### ISSUE TITLE
Does not always find the latest

### ISSUE THREAD

**Issue by john-kurkowski** [jkoenig134/gha-check-updates#1]
Hey love this tool! Surprised no one else has done it.

I ran `npx gha-check-updates` out on the following example workflow from [Lost Pixel](https://github.com/lost-pixel/lost-pixel)'s onboarding. I found gcu wasn't always finding the latest tag of a project. 

# Expected

```
  Action 'actions/checkout@v3' has a newer version available: 'v4' or 'v4.1.2'.
  Action 'actions/setup-node@v3' has a newer version available: 'v4' or 'v4.0.2'.
  Action 'lost-pixel/lost-pixel@v3' has a newer version available: 'v3' or 'v3.16.0'.
```

# Actual

```
  Action 'actions/checkout@v3' has a newer version available: 'v4' or 'v4.1.2'.
  Action 'actions/setup-node@v3' has a newer version available: 'v4' or 'v4.0.2'.
  Action 'lost-pixel/lost-pixel@v3' has a newer version available: 'undefined' or 'v3.9.0'.
```

# Workflow example provided by Lost Pixel's onboarding

```yml
// .github/workflows/vrt.yml
on: [push]

jobs:
  build:
    runs-on: ubuntu-latest
    name: Lost Pixel

    steps:
      - name: Checkout
        uses: actions/checkout@v3

      - name: Setup Node
        uses: actions/setup-node@v3
        with:
          node-version: 16.x

      - name: Install dependencies
        run: npm install

      - name: Build app
        run: npm run build

      - name: Start server (in background)
        run: npm run start &

      - name: Lost Pixel
        uses: lost-pixel/lost-pixel@v3.4.0
        env:
          LOST_PIXEL_API_KEY: ${{ secrets.LOST_PIXEL_API_KEY }}
```

**Comment by john-kurkowski** [comment]
The project's Git command to lookup tags does list the expected tags, as of today:

```sh
$ git -c 'versionsort.suffix=-' ls-remote --tags --sort='v:refname' https://github.com/lost-pixel/lost-pixel.git | tail
e848b0157a2f9ad40af910b9a4eb5ab3908723f2	refs/tags/v3.13.0
3ac2e30343c259397f211cee9747695a86500c33	refs/tags/v3.13.0^{}
3830cbdc45905ccb894574f90ee73158788fa5be	refs/tags/v3.14.0
f6f193259fa03946b997016d856cb3a9ce257563	refs/tags/v3.14.0^{}
e83bd4b7e4c3fca68e62f8c91715ef1f883ffbcf	refs/tags/v3.15.0
d9c7669e1568c92d2bd18fb7101b7f8f4b2d5fd6	refs/tags/v3.15.0^{}
9a7daf93fa4f59f03d7017b64248ee22c7ee39af	refs/tags/v3.15.1
9b34e46400f3f0f6d84308da9f9920335f2486f6	refs/tags/v3.15.1^{}
d3757094ba5fe1e15e88e29ff3a01bc172409f0b	refs/tags/v3.16.0
6db31948aef97f10ea9c7b1a8fa8dd3a0ce2dd49	refs/tags/v3.16.0^{}
```

Looks like gcu makes some tag name length assumptions, like [this line](https://github.com/jkoenig134/gha-check-updates/blob/78126323e1e151aa7412b4c490c475f418f747b9/src/index.ts#L72)?

**Comment by jkoenig134** [comment]
Hi @john-kurkowski! Thanks for the issue.
I published version 1.0.7 of the tool, which should solve your problems!

**Comment by john-kurkowski** [comment]
❤️ Thank you!
